docs/reference-guides/block-api/block-patterns.md

```diff
@@ -14,7 +14,7 @@ In this Document:
 
 The editor comes with several core block patterns. Theme and plugin authors can register additional custom block patterns using the `register_block_pattern` helper function.
 
-The `register_block_pattern` helper function receives two arguments. 
+The `register_block_pattern` helper function receives two arguments.
```

Comment:
There are some auto formatting changes that removed some extra spaces like this one. The main changes are in `blockTypes` and `contextual patterns` sections